app/routes.js

```diff
@@ -1,13 +1,18 @@
 // @flow
 import React from 'react';
-import { Route } from 'react-router';
+import { BrowserRouter as Router } from 'react-router-dom';
```

Comment:
Related docs https://reacttraining.com/react-router/web/api/Router, we can use BrowserRouter or HashRouter from 'react-router-dom'. Docs about BrowserRouter https://reacttraining.com/react-router/web/api/BrowserRouter

The `{ BrowserRouter as Router }` is simply importing and renaming it to Router, that we can use `<Router>` later on, instead of `<BrowserRouter>`. This is optional we could use `<BrowserRouter>` if you prefer that.